### DISCUSSION TITLE
Contributing to Documentation

### DISCUSSION THREAD

**Discussion by splashsky** [Annoraaq/grid-engine#49] [General]
Hey Anno, is there anywhere/anyway I and others can help contribute to documentation? I'd love to contribute in some way, but unfortunately I'm only just learning JS/TS (thanks to your plugin, no less). I write some mean English, though

**Comment by Annoraaq** [comment]
Hi splashsky, it is great to hear that you are willing to contribute.

I am planning on integrating gitbook with github, so you can simply create a pull request on github for the documentation.

**Reply by Annoraaq** [reply]
Or maybe I will just switch completely to Github Wiki. That makes it even easier

**Reply by Annoraaq** [reply]
@splashsky I just moved the documentation to the wiki of this repo. You should now be able to contribute.

**Reply by splashsky** [reply]
Unfortunately it looks like you can't make PRs for the Wiki feature 😞

At best maybe I could push directly to the wiki if I had Push authority? I leave the solution up to you. In the meantime, you can preview the changes I make at my fork of the repo; https://github.com/splashsky/phaser-grid-movement-plugin/wiki

**Reply by splashsky** [reply]
Actually, it may be best if we put the documentation into the GitHub Pages site for the repo. It would allow PRs for the docs and avoid the trouble of trying to merge changes into the Wiki feature, and avoid maintaining a separate docs repo. I'll submit a PR with some Pages adjustments. 😄

**Reply by Annoraaq** [reply]
I also thought about that. Great idea. I am looking forward to the PR!

**Comment by splashsky** [comment]
Just as an update, I am still working on the docs update; I'll be publishing it to the `docs/` folder in the repo, which we can point GitHub Pages to for documentation. As of writing this I'm converting all the examples to the new layout.

Here's the basic example and a preview of the docs.
![image](https://user-images.githubusercontent.com/12104206/110842374-a0c0c980-826c-11eb-8c94-ea5ad48a34be.png)

**Reply by Annoraaq** [reply]
Thank you for your contributions 🙂 
I am looking forward to it. The screenshot looks great already.

**Comment by KShivendu** [comment]
Hey @Annoraaq @splashsky , where can I find the assets used in examples ?

For example : 
In https://annoraaq.github.io/grid-engine/examples/basic-movement.html: 
- `assets/cloud_tileset.png`
- `assets/cloud_city.json`
- `assets/characters.png`

are required but I couldn't find these anywhere in the repository.

**Reply by Annoraaq** [reply]
Hey @KShivendu,

you can find those on our docs branch:

https://github.com/Annoraaq/grid-engine/tree/gh-pages/examples/assets

I hope this helps :)
